run.py

```diff
@@ -0,0 +1,53 @@
+import os.path as path
+import getopt, sys
+import src.argument_parser as parser
+
+args = parser.parse()
```

Comment:
Sorry to ask, but have you considered to use internal python lib [argparse](https://docs.python.org/3/library/argparse.html) instead of re implementing this?